bsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@godu/bs-ava",
-  "version": "3.0.0",
+  "name": "@ava/bucklescript",
+  "version": "0.0.0",
```

Comment:
Given that the tags and releases carried over with the repo move, I'd be happy to increment from `3.0.0.` onwards.